tests/inputs/HELIOTRON_vacuum

```diff
@@ -27,7 +27,7 @@ spectral_indexing = ansi
 node_pattern      = jacobi
 
 # pressure and rotational transform/current profiles
-l:   0	p =   0    i = 0
+l:   0	p =   0    i = 1E-6
```

Comment:
It might not be relevant to what you're testing, but I had to debug something on this file on #556 and ended up adding the below note to this file so we don't forget.
```python
# Note that the vacuum objective will optimize for current density = 0.
# The rotational transform profile coefficients below will only be used
# as an initial guess, as the rotational transform will not be constrained
# throughout the optimization. See GitHub issue #278.
```

Reply:
I am trying to resolve the issue that iota_flag, pres_flag, and curr_flag are True even when the coefficients are all zeros. This gives the following warning 
```
           warnings.warn(
                "Vacuum objective does not use any profiles, "
                + "ignoring presssure, iota, and current"
```
, which fails my tests. The warning shouldn't arise if all the coefficients are zero. So I change that part of parse_inputs().

For the Heliotron, one of the tests checks if there is a warning when we specify iota coefficients. To make sure that test doesn't fail, I made the iota coefficient finite.

Reply:
This is expected behavior. `pres_flag`, `iota_flag` etc are true if the given profiles are present in the input file, even if they're all zero. If the code throws a warning that we expect it to throw, you can wrap the part of the test like so:
```python
with pytest.warns(UserWarning):
    some code that throws warning
```

That will make sure the warning is thrown when we expect it to be, but not fail the test.

Reply:
I did not change that part of the code in the latest commit. I use with pytest.warns now.